pycqed/instrument_drivers/physical_instruments/DDM/DDMq.py

```diff
@@ -0,0 +1,318 @@
+                                   self.getTVdata, ch_pair)
+
+                               # vals=vals.Numbers(-128,127)
+                               )
+            scaladc_cmd = 'qutech:caladc{}'.format(ch_pair)
+            self.add_parameter('ch_pair{}_cal_adc'.format(ch_pair),
+                               label=('Calibrate ADC{}'.format(ch_pair)),
+                               #get_cmd=snsamp_cmd + '?',
+                               set_cmd=scaladc_cmd
+                               # vals=vals.Numbers(1,4096)
+                               )
+            self.add_parameter('ch_pair{}_qstate_cnt_data'.format(ch_pair),
+                               label=('get qstate counter' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=self._gen_ch_get_func(
+                                   self.getQstateCNT, ch_pair),
+
+                               # vals=vals.Numbers(-128,127)
+                               )
+            self.add_parameter('ch_pair{}_qstate_avg_data'.format(ch_pair),
+                               label=('get qstate average' +
+                                      'ch_pair {} '.format(ch_pair)),
+                               get_cmd=self._gen_ch_get_func(
+                                   self.getQstateAVG, ch_pair),
+
+                               # vals=vals.Numbers(-128,127)
+                               )
+        for i in range(self.device_descriptor.numChannels):
+            ch = i+1
+            self.add_parameter('ch{}_inavg_data'.format(ch),
+                               label=('get data ch {} '.format(ch)),
+                               get_cmd=self._gen_ch_get_func(
+                                   self.getInputAverage, ch),
+
+                               vals=vals.Numbers(-128, 127)
+                               )
+            srotres_cmd = 'qutech:wintrot{}:result'.format(ch)
+            self.add_parameter('ch{}_wintrot_result'.format(ch),
+                               units='#',
+                               label=('rotated integration result of' +
+                                      'channel {} '.format(ch)),
+                               get_cmd=srotres_cmd + '?'
+                               # vals=vals.Numbers(-134217728,134217727)
+                               )
+            sthl_cmd = 'qutech:qstate{}:threshold'.format(ch)
+            self.add_parameter('ch{}_qstate_threshold'.format(ch),
+                               units='#',
+                               label=('set threshold of ch {} '.format(ch)),
+                               get_cmd=sthl_cmd + '?',
+                               set_cmd=sthl_cmd + ' {}'
+                               # vals=vals.Numbers(-134217728,134217727)
+                               )
+            self.add_parameter('ch{}_weight_data'.format(ch),
+                               label=('get weigtht data ch {} '.format(ch)),
```

Comment:
This label does not describe what data this is, to clarify, if we would plot this we would not put this on the axis label. 
I think a better label would be : `Integration weigths channel {}`